QUARKUS-1027.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-1027 OpenShift Service Binding - Tech preview
+
+Jira: https://issues.redhat.com/browse/QUARKUS-1027
+
+The goal of this feature is to provide technology preview support of OpenShift Service Binding for RHOSAK in Quarkus 
+applications.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* None. The testing is executed manually for the Dragonball release by deploying Quarkus Kafka Quickstart in OpenShift 
```

Comment:
Dragonball => RHBQ 2.2

Reply:
I am missing the features that are going to be covered. 
For example, 
- Deployment of a Quarkus app on openshift 4.6 by S2I that use an Strimzi Kafka with registry

Reply:
We won't be running this on 4.6, service binding is primarily aimed at OSD

Reply:
`Dragonball => RHBQ 2.2` not yet addressed

Reply:
Fixed :)

@pjgg Is the wording better now?